### PR TITLE
DFASTMI-139: modify GUI layout

### PR DESCRIPTION
**Pull request by ralph-peelen-deltares**

### CODE REVIEW

File: dfastmi/gui/dialog_view_model.py [hunk 1]
```diff
@@ -87,7 +95,8 @@ def output_dir(self):
     @output_dir.setter
     def output_dir(self, value):
         self._output_dir = value
-        self.model.section["OutputDir"] = value
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view_model.py:90]: Why were all the extra checks added? We know that the view model has a `model` - it is never None, so no need to check it everywhere.

File: dfastmi/gui/dialog_view.py [hunk 2]
```diff
@@ -102,37 +105,49 @@ def update_reach(self, data):
...
-        for condition_tab_index, measure_file in enumerate(self.view_model.measure_files):
-            tab = self._tabs.widget(condition_tab_index +1)
-            if tab:
-                prefix = str(condition_tab_index)+"_"
-                key = prefix +"file2"
-                input_textbox = tab.findChild(QtWidgets.QLineEdit, key)
-                if input_textbox:
-                    input_textbox.setText(measure_file)
+        for condition_discharge, measure_file in self.view_model.measure_files.items():        
+            prefix = str(condition_discharge)+"_"
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:116]: These two for-loops can easily be deduplicated. 

```
    def update_condition_files(self):
        for condition_discharge, reference_file in self.view_model.reference_files.items():
            self._update_condition_file_field("file1", condition_discharge, reference_file)
        
        for condition_discharge, measure_file in self.view_model.measure_files.items():        
            self._update_condition_file_field("file2", condition_discharge, measure_file)
    
    def _update_condition_file_field(self, field_postfix: str, condition_discharge: float, condition_file: str) -> None:
            prefix = str(condition_discharge) + "_"
            key = prefix + field_postfix
            input_textbox = self.general_widget.findChild(QtWidgets.QLineEdit, key)
            if input_textbox:
                input_textbox.setText(condition_file)
```

File: dfastmi/gui/dialog_view.py [hunk 3]
```diff
@@ -102,37 +105,49 @@ def update_reach(self, data):
...
+            key = prefix +"file2"
+            input_textbox = self.general_widget.findChild(QtWidgets.QLineEdit, key)
+            if input_textbox:
+                input_textbox.setText(measure_file)
     
     def update_qvalues_tabs(self):
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:122]: Update function name

File: dfastmi/gui/dialog_view.py [hunk 4]
```diff
@@ -102,37 +105,49 @@ def update_reach(self, data):
...
-        tabs = self._tabs
-        for j in range(tabs.count()-2,-1,-1):
-            tabs.removeTab(1+j)            
+        self._clear_conditions()
+        for discharge in self.view_model.current_reach.hydro_q:
+            prefix = str(discharge) + "_"
+            qval = str(discharge) + " m3/s"
+            self.add_condition_line(prefix, discharge, qval)
+
+    def _clear_conditions(self):
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:129]: I think it would be easier and more readable to just initialize a new empty `grid_layout`..

File: dfastmi/gui/dialog_view.py [hunk 5]
```diff
@@ -260,10 +279,50 @@ def add_general_tab(self, tabs: PyQt5.QtWidgets.QTabWidget, win: PyQt5.QtWidgets
...
+        layout.addRow(group_box)        
+
         # get the output directory
         self.output_dir = QtWidgets.QLineEdit(win)
-        self.output_dir.textChanged.connect(self.output_dir_text_changed)
-        layout.addRow(self.view_model.gui_text("outputDir"), self.openFileLayout(win, self.output_dir, "output_dir"))
+        # Create a file validator to check if the file exists
+        file_validator = FileExistValidator()
+        self.output_dir.setValidator(file_validator)
+        #self.output_dir.editingFinished.connect
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:323]: Remove commented out code.

File: dfastmi/gui/dialog_view.py [hunk 6]
```diff
@@ -284,27 +343,51 @@ def add_general_tab(self, tabs: PyQt5.QtWidgets.QTabWidget, win: PyQt5.QtWidgets
...
+            self.view_model.ucritical = float(self.ucrit.text())
+            self.update_qvalues_tabs()
+        else: 
+            self.showMessage("Please input valid values for ucritical")
+    
+    
+    def output_dir_text_changed(self):
+        state = self.output_dir.validator().validate(self.output_dir.text(), 0)[0]
+        if state == PyQt5.QtGui.QValidator.Acceptable:
+            self.showMessage("File exists!")
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view.py:376]: This is not necessary

File: dfastmi/gui/dialog_view_model.py [hunk 7]
```diff
@@ -114,7 +125,8 @@ def save_plots(self):
     @save_plots.setter
     def save_plots(self, value):
         self._save_plots = value
-        self.model.section["SavePlots"] = str(value)
+        if self.model and self.model.section :
+            self.model.section["SavePlots"] = str(value)
```
Comment (priscavdsluis) [inline dfastmi/gui/dialog_view_model.py:129]: I don't understand why we have a local field ánd the same data on the model for each property.

**Comment by priscavdsluis** [comment]
I wonder if we need to let go of the `General` tab, since we only have one tab now and I don't see that changing anytime. Will make the GUI look a lot cleaner.

File: dfastmi/gui/dialog_view.py (hunk 2 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:134]: good idea, updated

File: dfastmi/gui/dialog_view.py (hunk 3 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: done

File: dfastmi/gui/dialog_view.py (hunk 5 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: done

File: dfastmi/gui/dialog_view.py [hunk 8]
```diff
@@ -284,27 +343,51 @@ def add_general_tab(self, tabs: PyQt5.QtWidgets.QTabWidget, win: PyQt5.QtWidgets
...
+        if self.ucrit.hasAcceptableInput():
+            self.view_model.ucritical = float(self.ucrit.text())
+            self.update_qvalues_tabs()
+        else: 
+            self.showMessage("Please input valid values for ucritical")
+    
+    
+    def output_dir_text_changed(self):
+        state = self.output_dir.validator().validate(self.output_dir.text(), 0)[0]
+        if state == PyQt5.QtGui.QValidator.Acceptable:
```
[the remark this answers is not in the document]
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: code was unneeded, removed

File: dfastmi/gui/dialog_view.py (hunk 6 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: agreed, removed

File: dfastmi/gui/dialog_view_model.py (hunk 1 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view_model.py:90]: good idea! will update

File: dfastmi/gui/dialog_view_model.py (hunk 7 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view_model.py:129]: I agree, this needs to be resolved

File: dfastmi/gui/dialog_view.py (hunk 4 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:149]: I would love a new implementation.... but i don't know how... sorry! please help me :-)

File: dfastmi/gui/dialog_model.py [hunk 9]
```diff
@@ -122,17 +122,21 @@ def _get_condition_configuration(self, config : ConfigParser, reach : AReach, re
...
-                config[cond]["Reference"] = reference_files[i]
-            else:
-                config[cond]["Reference"] = ""  # Default value if index is out of range
-
-            if i < len(measure_files):
-                config[cond]["WithMeasure"] = measure_files[i]
-            else:
-                config[cond]["WithMeasure"] = ""  # Default value if index is out of range
+        i = 0
+        for discharge in enumerate(reach.hydro_q[:num_files]): # Ensure it loops until the minimum length
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_model.py:126]: Maybe move this to a descriptive method?
E.g.
set_all_discharges_references_measures

File: dfastmi/gui/dialog_utils.py [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+            return (PyQt5.QtGui.QValidator.Acceptable, input_str, pos)
+        else:
+            return (PyQt5.QtGui.QValidator.Invalid, input_str, pos)
+
+
+class ValidatingLineEdit(QtWidgets.QLineEdit):
+    def __init__(self, validator:PyQt5.QtGui.QValidator=FileExistValidator(), parent=None):
+        super().__init__(parent)
+        self.validator = validator
+        #self.setValidator(self.validator)
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_utils.py:54]: should this be commented?

File: dfastmi/gui/dialog_utils.py [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+    def setInvalid(self, invalid):
+         self.invalid = invalid
+         self.update()
+
+    def paintEvent(self, event):
+        super().paintEvent(event)
+        if self.isEnabled():
+            if self.invalid:
+                painter = PyQt5.QtGui.QPainter(self)
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_utils.py:65]: cant this be moved to a method and that you only pass in the color?

File: dfastmi/gui/dialog_utils.py [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+                painter.end()  # Ensure to end the painter
+            else:
+                painter = PyQt5.QtGui.QPainter(self)
+                painter.setPen(PyQt5.QtCore.Qt.green)
+                painter.setBrush(PyQt5.QtCore.Qt.NoBrush)
+                painter.drawRect(PyQt5.QtCore.QRect(0, 0, self.width() - 1, self.height() - 1))
+                painter.end()  # Ensure to end the painter
+
+    def validate(self, input_str, pos):
+        state, _, _ = self.validator.validate(input_str, pos)
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_utils.py:78]: should the last two returned variables be discarded?

Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_utils.py:78]: maybe make the name of the validator more specific?
You now use a validator in the validator to validate some validation.

File: dfastmi/gui/dialog_utils.py [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+            return state, input_str, pos
+        else:
+            return state, input_str[:pos], pos
+
+
+def get_available_font(currrent_font : PyQt5.QtGui.QFont, preferred_font, fallback_font):
+    # Check if the preferred font is available
+    available_fonts = PyQt5.QtGui.QFontDatabase().families()
+
+    if preferred_font in available_fonts:
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_utils.py:89]:
```
    if preferred_font in available_fonts:
        return PyQt5.QtGui.QFont(preferred_font, currrent_font.pointSize())

    if fallback_font in available_fonts:
        return PyQt5.QtGui.QFont(fallback_font, currrent_font.pointSize())

    return currrent_font
```

Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_utils.py:89]: what is the exact difference between fallback font and current font?

File: dfastmi/gui/dialog_view.py [hunk 14]
```diff
@@ -142,235 +175,324 @@ def create_qt_application(self) -> None:
         None
         """
         
-        self._app = QtWidgets.QApplication(sys.argv)
+        self._app = QApplication(sys.argv)
         self._app.setStyle("fusion")
+        
+        # Set the application-wide font
+        font = self._app.font()
+        preferred_font = "Lucida Console" #"Consolas"
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:183]:
```suggestion
        preferred_font = "Lucida Console"
```

File: dfastmi/gui/dialog_view.py (hunk 4 above)
Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_view.py:146]: What are you exactly trying to do here, because it feels like you try to do something that should feels like it would be a standard funtion of widget. (that being said, I am no QT expert, so I have no actual footing on my claim)
Maybe we can look into making this method more readable and less indentable.

File: dfastmi/gui/dialog_view.py [hunk 15]
```diff
@@ -142,235 +175,324 @@ def create_qt_application(self) -> None:
...
-            self.view_model.output_dir = text
+        self._create_make_plots_input_checkbox(layout)
+        self._create_save_plots_input_checkbox(layout)
+        self._create_output_figure_plots_directory_input(layout)        
+        self._create_close_plots_input_checkbox(layout)
+        
+        self._layout.addWidget(self._general_widget)
+
+    def _create_close_plots_input_checkbox(self, layout):
+        self._close_plots = QLabel(self._view_model.gui_text("closePlots"), self._win)
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:280]: should all these texts be hardcoded in the gui?
Maybe something to discuss for a followup issue?

File: dfastmi/gui/dialog_view.py [hunk 16]
```diff
@@ -142,235 +175,324 @@ def create_qt_application(self) -> None:
...
+        self._branch.setCurrentText(self._view_model.current_branch.name)
+        layout.addRow(self._view_model.gui_text("branch"), self._branch)
+
+    def _update_qthreshold(self):
+        if self._qthr.hasAcceptableInput():
+            self._view_model.qthreshold = float(self._qthr.text())
+            self._update_qvalues_table()
+            self._update_condition_files()
+        else: 
+            self._showMessage("Please input valid values for qthreshold")
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:423]: This feels like something that should definitely not be hardcoded in the gui.
Lets look into a followup issue.

Same below.

File: dfastmi/gui/dialog_view.py [hunk 17]
```diff
@@ -390,81 +512,76 @@ def activate_dialog(self) -> None:
...
         Arguments
         ---------
         None
         """
-        msg = QtWidgets.QMessageBox()
-        msg.setIcon(QtWidgets.QMessageBox.Information)
+        msg = QMessageBox()
+        msg.setIcon(QMessageBox.Information)
         msg.setText("D-FAST Morphological Impact " + dfastmi.__version__)
         msg.setInformativeText("Copyright (c) 2024 Deltares.")
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:558]: Maybe we should also move these strings to something like a resource?
e.g. followup issue.

File: dfastmi/gui/dialog_view.py [hunk 18]
```diff
@@ -390,81 +512,76 @@ def activate_dialog(self) -> None:
...
 
         Arguments
         ---------
         None
         """        
-        subprocess.Popen(self.view_model.manual_filename, shell=True)
+        subprocess.Popen(self._view_model.manual_filename, shell=True)
     
-    def openFileLayout(self, win, myWidget, key: str):
+    def _openFileLayout(self, win, myWidget, key: str, enabled: bool):
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:584]: This feels the same as _openFolderLayout

File: dfastmi/gui/dialog_view_model.py [hunk 19]
```diff
@@ -105,7 +115,8 @@ def plotting(self):
     @plotting.setter
     def plotting(self, value):
         self._plotting = value
-        self.model.section["Plotting"] = str(value)
+        if self.model and self.model.section :
+            self.model.section["Plotting"] = str(value)
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view_model.py:119]: Would this dict be clashing with the PlotOptions class I introduced in 123? @ralph-peelen-deltares

**Comment by MRVermeulenDeltares** [comment]
Is it possible to put the model and/or the viewmodel under unit tests?

File: dfastmi/gui/dialog_view.py (hunk 4 above)
Comment (priscavdsluis) [reply dfastmi/gui/dialog_view.py:147]: In `create_qt_application` we setup a new grid layout (which is already empty). We can do the same here, instead of going through all of the columns/rows and clearing them. 
I can try to help with that if you like.

File: dfastmi/gui/dialog_view.py (hunk 14 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:181]: updated

File: dfastmi/gui/dialog_view.py (hunk 15 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: gui_text is from resource file _messages.LANGUAGE.ini_

File: dfastmi/gui/dialog_view.py (hunk 16 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: may be we need to remove this popup?

File: dfastmi/gui/dialog_view.py (hunk 17 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:560]: is followup issue

File: dfastmi/gui/dialog_view_model.py (hunk 19 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view_model.py]: no, but it got me thinking and updated in dialog_model.py

File: dfastmi/gui/dialog_view.py (hunk 18 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: yes they do! will look into this

File: dfastmi/gui/dialog_view.py (hunk 4 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:145]: Can we look at this in a new issue together? I can't get it as simple as you request

File: dfastmi/gui/dialog_view_model.py [hunk 20]
```diff
@@ -28,8 +28,9 @@
 """
 # ViewModel
 from configparser import ConfigParser
-import os
-from typing import Any, Dict
+from typing import Dict
+#from PyQt5 import QtCore
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view_model.py:32]: Remove if not used.

File: dfastmi/gui/dialog_view.py (hunk 17 above)
Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_view.py:560]: Is an issue already created?

File: pyproject.toml [hunk 21]
```diff
@@ -47,7 +49,7 @@ python_functions=["given_*",
                     "Given_*",
                     "test_*",
                     "Test_*"]
-
+#qt_api=pyqt5
```
Comment (MRVermeulenDeltares) [inline pyproject.toml:52]: remove if not used.

File: pyproject.toml [hunk 22]
```diff
@@ -28,6 +28,8 @@ Nuitka = "^1.3.8"
 imageio = "^2.33.1"
 pytest = "^7.2.1"
 pytest-cov = "^4.1.0"
+pytest-mock = "^3.0"
```
Comment (MRVermeulenDeltares) [inline pyproject.toml:31]: How is this mock different from the mock we previously used?

File: dfastmi/gui/dialog_utils.py (hunk 10 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py:57]: nope, updated

File: dfastmi/gui/dialog_utils.py (hunk 11 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py:68]: yes, thank you. Updated

File: dfastmi/gui/dialog_view.py (hunk 16 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: removed

File: dfastmi/gui/dialog_view.py (hunk 18 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: can't get it similar without a boolean switch... do you have thoughs? or ideas? let look into this later

File: dfastmi/gui/dialog_view_model.py (hunk 20 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view_model.py:32]: done

File: pyproject.toml (hunk 22 above)
Comment (ralph-peelen-deltares) [reply pyproject.toml:31]: needed to add this for the qt testing

File: pyproject.toml (hunk 21 above)
Comment (ralph-peelen-deltares) [reply pyproject.toml:52]: removed

**Comment by ralph-peelen-deltares** [comment]
> Is it possible to put the model and/or the viewmodel under unit tests? Sonarcloud shows 13 codesmells in dialog_view.py, please have a look at them and try to resolve them when possible.

Wrote all relevant tests



> I wonder if we need to let go of the `General` tab, since we only have one tab now and I don't see that changing anytime. Will make the GUI look a lot cleaner.
> 
> Please check SonarCloud as well.

Removed tab and processed all sonar cloud issues, except for _clear_conditions... I will create a new issue for that one

File: dfastmi/gui/dialog_view.py (hunk 4 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:148]: DFAST-185

File: dfastmi/gui/dialog_view.py (hunk 17 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py:559]: DFAST-186

File: dfastmi/gui/dialog_view.py (hunk 18 above)
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_view.py]: [DFAST-187](https://issuetracker.deltares.nl/browse/DFAST-187)

**Comment by ralph-peelen-deltares** [comment]
Feedback from @hrajagers on GUI:
1. Increase application width so input text box of withmeasure is same size as reference
2. Will check screenshot with RWS
3. GUI monospaced font only for reaches
4. discharge location in output of reference simulation groupbox should be correct (same as at std discharge location)
5. no need for green visual feedback
6. Graying out the fields under the “Minimum Flow-carrying Discharge” works fine. However, for the Lower Rhine (and Meuse) I see that the discharge under the Qstagnant is not grayed out. The “Minimum Flow-carrying Discharge” should actually always be greater than or equal to the Qstagnant of the selected branch.
7. saving configuration fails
8. icon must be provided in nuitka and in distribution or it will not be in the about box
9. impacted length is not always visualized properly

**Comment by ralph-peelen-deltares** [comment]
> Feedback from @hrajagers on GUI:
> 
> 1. Increase application width so input text box of withmeasure is same size as reference
> 2. Will check screenshot with RWS
> 3. GUI monospaced font only for reaches
> 4. discharge location in output of reference simulation groupbox should be correct (same as at std discharge location)
> 5. no need for green visual feedback
> 6. Graying out the fields under the “Minimum Flow-carrying Discharge” works fine. However, for the Lower Rhine (and Meuse) I see that the discharge under the Qstagnant is not grayed out. The “Minimum Flow-carrying Discharge” should actually always be greater than or equal to the Qstagnant of the selected branch.
> 7. saving configuration fails
> 8. icon must be provided in nuitka and in distribution or it will not be in the about box
> 9. impacted length is not always visualized properly

All open and possible failing feedback are implemented

File: dfastmi/gui/dialog_model.py [hunk 23]
```diff
@@ -71,74 +82,91 @@ def __init__(self, rivers_configuration: RiversObject, config_file: Optional[str
...
     @output_dir.setter
-    def output_dir(self, value):
+    def output_dir(self, value: str):
+        """Set output directory."""
         self.section['OutputDir'] = value
     
     @property
     def figure_dir(self) -> str:
+        """Get figure directory."""
         return self.section['FigureDir']
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_model.py:126]: is this related to the plotoptions class I setup in 123?

File: dfastmi/gui/dialog_view.py [hunk 24]
```diff
@@ -178,14 +239,8 @@ def _create_qt_application(self) -> None:
...
-        self._app.setStyle("fusion")
-        
-        # Set the application-wide font
-        preferred_font = "Lucida Console"
-        fallback_font = "Courier New"
-        font = get_available_font(self._app.font(), preferred_font, fallback_font, QFontDatabase())
-        self._app.setFont(font)
-
+        self._app.setStyle("fusion")        
+
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:243]: Font should no longer be set?

File: dfastmi/gui/dialog_view.py [hunk 25]
```diff
@@ -395,24 +558,55 @@ def _create_qthreshhold_input(self, layout, double_validator : PyQt5.QtGui.QDoub
...
+            None
+        """
         self._reach = QComboBox(self._win)
         self._reach.currentTextChanged.connect(self._view_model.updated_reach)
         self._reach.setToolTip(gui_text("reach_tooltip"))
         for r in self._view_model.current_branch.reaches:
             self._reach.addItem(r.name)
         self._reach.setCurrentText(self._view_model.current_reach.name)
+         # Set the reach font
+        preferred_font = "Lucida Console"
```
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_view.py:593]: shouldnt there be an overal font throughout the application?

File: dfastmi/gui/dialog_model.py (hunk 23 above)
Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_model.py:126]: discussed
